src/iced/top.iced

```diff
@@ -47,6 +48,7 @@ exports.run = run = ({passphrase, salt, progress_hook}, cb) ->
 
   out = generate seed_final
   out.seeds = seeds
+  out.mnemonic = mnemonic.fromSeed(seed_final.slice(0,16), mnemonic.Words.ENGLISH).toString()
```

Comment:
I was curious about this, so I asked on freenode#bitcoin. Someone pretty clever answered, so I've reproduced the relevant part of the conversation below.

In short, it seems like 24 word mnemonics are the way to go.

```
22:24:00 <rodarmor> Why does the ledger wallet use 24 word mnemonics by default?
Since the security of an 256 bit ECDSA key is 256/2  = 128, and 12 word mnemonic
has that much entropy, a 24 word mnemonic doesn't provide additional security.

22:30:53 <@smart-person> rodarmor: that is an overly simplistic analysis.

22:31:26 <rodarmor> @smart-person: I'm sure you're right. Can you elaborate a
little bit?

22:33:16 <@smart-person> There are a couple angles, one is one the straight
security claim.  if you do ~2^128 work you can crack an ecc key, but if you do
2^120 you're almost certan to have not cracked it (like one in a trillion levels).  While for 128 bit symmetric crypto if you do 2^120 work you have
roughly a 1% odds of having cracked it.

22:34:07 <@smart-person> The other is that the way users handle mnemonics may
result in people getting some of the words (e.g. seeing it over your shoulder,
or being able to pick up oils off a keyboard).. so that is an oppturnity to
leak data.

22:35:38 <@smart-person> I suppose third is that a BIP32 key actually has 512
bits of key material: 256 bits for the key itself and 256 bits for the
chaincode, and then you sometimes make public the chaincode-- so you're
potentially more exposed to weaknesses in the hash function.

22:36:39 <rodarmor> smart-person: Although a counter argument might be that if
the key is longer, it is more prone to accidental damage.

22:37:23 <rodarmor> But the three reasons you give are a much stronger argument
for using a longer key.

22:40:49 <@smart-person> also, another reason is that if a quantum computer is
constructed that is still too small to break ECC keys directly, it might still
be large enough to derrive a 128 bit seed from a public key (in 2^64 work, using
grover's algorithim)
```